Sources/CombineDataSources/CollectionView/CollectionViewItemsController.swift

```diff
@@ -97,6 +100,10 @@ public class CollectionViewItemsController<CollectionType>: NSObject, UICollecti
   public func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
     cellFactory(self, collectionView, indexPath, collection[indexPath.section][indexPath.row])
   }
+
+  public func collectionView(_ collectionView: UICollectionView, viewForSupplementaryElementOfKind kind: String, at indexPath: IndexPath) -> UICollectionReusableView {
+      return configureSupplementaryView!(self, collectionView, kind, indexPath, collection[indexPath.section])
```

Comment:
configureSupplementaryView is optional, right? do you need to force unwrap it here?

Reply:
Yes because the collection view delegate method requires a returned supplementary view. Unless we were to guard it and cause a fatal error with a message saying the property must not be nil